internal/hcsoci/resources_lcow.go

```diff
@@ -163,7 +163,7 @@ func allocateLinuxResources(ctx context.Context, coi *createOptionsInternal, r *
 	addGPUVHD := false
 	for i, d := range coi.Spec.Windows.Devices {
 		switch d.IDType {
```

Comment:
This seems like another case where we should error if we see unsupported devices.

Reply:
Done :)